.gitlab-ci.yml

```
@@ -155,7 +175,32 @@ include:
           echo "WARNING: GitHub CLI (gh) not found in after_script, trying to continue anyway"
         fi
 
+        error_log_file="sorc/logs/error.logs"
         if [[ "${GFS_CI_RUN_TYPE}" == "pr_cases" && "${PR_NUMBER}" != 0 ]]; then
+          if [[ -f "${error_log_file}" ]]; then
+            pr_error_log_links=$(dev/ci/scripts/utils/ci_utils.sh publish_logs_from_file "PR_${PR_NUMBER}" "${error_log_file}")
+          else
+            pr_error_log_links="(no error.logs file found)"
+          fi
+
+          if [[ -f "${error_log_file}" && -s "${error_log_file}" ]]; then
```

Nitpick: The `-f` test isn't necessary here.  `-s` also tests for existence.

```suggestion
          if [[ -s "${error_log_file}" ]]; then